DiTails/UI/DetailViewHost.cs

```diff
@@ -131,8 +131,8 @@ private void SetupVotingButtons()
             {
                 if (votingUpvoteImage != null && votingDownvoteImage != null)
                 {
-                    votingUpvoteImage.SetImage("DiTails.Resources.arrow.png");
-                    votingDownvoteImage.SetImage("DiTails.Resources.arrow.png");
+                    votingUpvoteImage.SetImageAsync("DiTails.Resources.arrow.png");
+                    votingDownvoteImage.SetImageAsync("DiTails.Resources.arrow.png");
```

Comment:
These asynchronous method are not `await`ed.

Reply:
I'm not sure what you mean by this

Reply:
Calling method does not appear to be async
![image](https://github.com/Auros/DiTails/assets/42526028/7c9a5104-f263-4a90-9af3-bf3f8f83dfa1)

Reply:
Should probably be something like this?
```
Index: DiTails/UI/DetailViewHost.cs
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/DiTails/UI/DetailViewHost.cs b/DiTails/UI/DetailViewHost.cs
--- a/DiTails/UI/DetailViewHost.cs	(revision 64b8ecbfd0f5154e167dffb0dc25786bbf1f2808)
+++ b/DiTails/UI/DetailViewHost.cs	(date 1713994715401)
@@ -126,14 +126,14 @@
             }
         }
 
-        private void SetupVotingButtons()
+        private async Task SetupVotingButtons()
         {
             if (!_didSetupVote)
             {
                 if (votingUpvoteImage != null && votingDownvoteImage != null)
                 {
-                    votingUpvoteImage.SetImage("DiTails.Resources.arrow.png");
-                    votingDownvoteImage.SetImage("DiTails.Resources.arrow.png");
+                    await votingUpvoteImage.SetImageAsync("DiTails.Resources.arrow.png");
+                    await votingDownvoteImage.SetImageAsync("DiTails.Resources.arrow.png");
                     votingUpvoteImage.DefaultColor = new Color(0.388f, 1f, 0.388f);
                     votingDownvoteImage.DefaultColor = new Color(1f, 0.188f, 0.188f);
 
@@ -177,7 +177,7 @@
         {
             _activeBeatmap = difficultyBeatmap;
             await Parse(standardLevelDetailViewController);
-            SetupVotingButtons();
+            await SetupVotingButtons();
 
             ShowPanel = false;
             parserParams?.EmitEvent("show-detail");
```